rachellegelden/Home-Together-Canada
# HTC-151: setup MySQL DB and added abstractUser model

# [151](https://github.com/rachellegelden/Home-Together-Canada/issues/151)

## Summary
- setup local MySQL database and configuration for development
- setup local test database (for integration tests that are to come)
- setup remote MySQL database on Heroku and added configuration to express app for after the app is deployed
- added a model for abstractUser and controller to create user and do basic searches

## Relevant Motivation & Context
Required to support current front-end development

## Testing Instructions
Install MySql and run it locally (you may need to change the username and password information in the dbConfig file). After its up and running, you can use an API testing software such as postman to ping the server to the routes in routes.js and you will be able to see the additions to the database as well as the results from your queries. Note if you use Postman, get the desktop app and not the in-browser app

## Developer checklist prior to opening this pull request:


## Reviewer
